airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3778,19 +3778,47 @@ components:
           type: string
     OperatorWebhook:
       type: object
-      required:
-        - executionUrl
       properties:
-        executionUrl:
-          type: string
-          description: The URL to call to execute the webhook operation via POST request.
-        executionBody:
-          type: string
-          description: If populated, this will be sent with the POST request.
         webhookConfigId:
           type: string
           format: uuid
           description: The id of the webhook configs to use from the workspace.
+        webhookType:
+          type: string
+          enum:
+            - generic
+            - dbtCloud
+        generic:
+          type: object
+          required:
+            - executionUrl
+          properties:
+            executionUrl:
+              type: string
+              description: The URL to call to execute the webhook operation via POST request.
+            executionBody:
+              type: string
+              description: If populated, this will be sent with the POST request.
+        dbtCloud:
+          type: object
+          required:
+            - accountId
+            - jobId
+          properties:
+            accountId:
+              type: integer
+              description: The account id associated with the job
+            jobId:
+              type: integer
+              description: The job id associated with the job
```

Comment:
Using the same field names and types is the key thing to make the frontend DX smooth :+1:. Using the same type through the whole "list -> select -> submit" pipeline would let the frontend code be even cleaner, but not by a big enough margin to justify a lot of backend hassle.